src/main/java/org/cubeville/cvgames/models/GameQueue.java

```diff
@@ -316,6 +347,14 @@ public ItemStack queueLeaveItem() {
 		return GameUtils.customItem(Material.RED_BED, "§c§l§oLeave Queue §7§o(Right Click)");
 	}
 
+	public ItemStack spectatorLeaveItem() {
+		return GameUtils.customItem(Material.RED_BED, "§c§l§oLeave §7§o(Right Click)");
+	}
+
+	public ItemStack playerCompassItem() {
+		return GameUtils.customItem(Material.COMPASS, "§e§l§oPlayer Compass §7§o(Right Click)");
+	}
+
 	public ItemStack teamSelectorItem() {
 		return GameUtils.customItem(Material.GHAST_TEAR, "§b§l§oSelect Team §7§o(Right Click)");
 	}
```

Comment:
Are you interested at all in making fancy hex-color names for the items defined here? The offer is open to you if you want, otherwise, I'll leave it be.